pilot/cmd/pilot-agent/status/server.go

```diff
@@ -206,9 +209,9 @@ func (s *Server) handleAppProbe(w http.ResponseWriter, req *http.Request) {
 	}
 	var url string
 	if prober.Scheme == corev1.URISchemeHTTPS {
-		url = fmt.Sprintf("https://localhost:%v%s", prober.Port.IntValue(), prober.Path)
```

Comment:
why do you need to change this?

Reply:
The app liveness/readiness probe may also have the same issue.